static/bidder-info/nativo.yaml

```diff
@@ -0,0 +1,22 @@
+endpoint: "https://exchange.postrelease.com/esi?ntv_epid=7"
+maintainer:
+  email: "prebiddev@nativo.com"
+gvlVendorID: 263
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
+userSync:
+  redirect:
+    url: https://jadserve.postrelease.com/suid/101787?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&ntv_gpp_consent={{.GPP}}&ntv_r={{.RedirectURL}}
+    userMacro: NTV_USER_ID
```

Comment:
Confirmed redirect user sync is working. Observed 302 to PBS setuid endpoint with a user ID which is successfully added to cookie.
<img width="1059" alt="Screenshot 2024-11-08 at 9 09 40 AM" src="https://github.com/user-attachments/assets/65121c81-3259-41e5-8611-58e55680fda2">